### ISSUE TITLE
CSS causes issue with Turbopack 

### ISSUE THREAD

**Issue by jovermier** [Leaflet/Leaflet#9261]
### Checklist

- [X] I've looked at the [documentation](https://leafletjs.com/reference.html) to make sure the behavior isn't documented and expected.
- [X] I'm sure this is an issue with Leaflet, not with my app or other dependencies (Angular, Cordova, React, etc.).
- [X] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this hasn't been reported yet.
- [X] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Steps to reproduce

Use Turbopack to build the project. Observe the build fails to parse the leaflet CSS.

The issue was closed on turbopack's repo siting it is an upstream issue in leaflet. https://github.com/vercel/turbo/issues/2451

### Expected behavior

The build should not fail to parse the leaflet CSS.

### Current behavior

Turbopack fails to build
```
error - [parse] [project-with-next]/node_modules/.pnpm/leaflet@1.8.0/node_modules/leaflet/dist/leaflet.css  [project-with-next]/node_modules/.pnpm/leaflet@1.8.0/node_modules/leaflet/dist/leaflet.css:552:32  Parsing css source code failed
     548   	width: 24px;
     549   	margin: 0 auto;
     550   
     551   	-ms-filter: "progid:DXImageTransform.Microsoft.Matrix(M11=0.70710678, M12=0.70710678, M21=-0.70710678, M22=0.70710678)";
     552 > 	filter: progid:DXImageTransform.Microsoft.Matrix(M11=0.70710678, M12=0.70710678, M21=-0.70710678, M22=0.70710678);
     553   	}
     554   
     555   .leaflet-oldie .leaflet-control-zoom,
     556   .leaflet-oldie .leaflet-control-layers,
  
  Expected Declaration value
  ```

### Environment

- Leaflet version: 1.9.4
- Browser (with version): na
- OS/Platform (with version): linux

**Comment by mourner** [comment]
Looks like this was originally added in 2011 to make popup tip IE6–8 properly displayed (rotated 45 degrees for the "tip" appearance). Not sure if we want to care about this anymore cc @Falke-Design. Might consider removing the offending line (or both filter lines) for v1.9.5 — popups should remain accessible even in old IE.

**Comment by Falke-Design** [comment]
In `v2` it would not apply because we already removed that we set the `leaflet-oldie` class. So yes we should remove the filters with `v2` too.

https://github.com/Leaflet/Leaflet/blob/83fb5a4226e7c306631b3af7d3c5c96cc9a702f3/src/map/Map.js#L1106-L1112

Also it looks like Leaflet doesn't work with IE6-8 anymore.
IE6 until IE8 have the same error:
![grafik](https://github.com/Leaflet/Leaflet/assets/19800037/02584049-bda4-46ca-a022-fd8cf508f1cd)

With Edge simulation of IE8 I get the following error:
> ':' expected

![grafik](https://github.com/Leaflet/Leaflet/assets/19800037/6be7994b-7fee-4b94-88cc-1e28f5a0c672)

which points to `get lastId()`. The object looks like the `Util.js` class
```
var tt = {
    __proto__: null,
    extend: l,
    create: R,
    bind: a,
    get lastId() {
        return D
    },
    stamp: h,
    throttle: j,
    wrapNum: H,
    falseFn: u,
    formatNum: i,
    trim: W,
    splitWords: F,
    setOptions: c,
    getParamString: U,
    template: q,
    isArray: d,
    indexOf: G,
    emptyImageUrl: K,
    requestFn: $,
    cancelFn: Q,
    requestAnimFrame: x,
    cancelAnimFrame: r
};
```

https://github.com/Leaflet/Leaflet/blob/83fb5a4226e7c306631b3af7d3c5c96cc9a702f3/src/core/Util.js#L50


So in my eyes we can drop support for IE8 and lower with 1.9.5 because it already doesn't work anymore.

**Comment by mourner** [comment]
@Falke-Design yeah, no one really cares about those versions, and even Leaflet website states IE9+ support on the website for quite some time now. Let's just remove that stuff for v1.9.5.

**Comment by jonkoops** [comment]
+1 let's throw this in the bin.

**Comment by jonkoops** [comment]
@mujypatahy8 stop spamming AI generated garbage or I will have you banned.

**Comment by AshwinNema** [comment]
Hi,
I am new to open source. Can I take up this issue?

**Comment by RegalFenster** [comment]
This is already fixed https://github.com/Leaflet/Leaflet/pull/8559?

**Comment by jonkoops** [comment]
Correct, this has been fixed. Closing.
